mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/impl/PojoMassIndexingLoggingMonitor.java

```diff
@@ -53,6 +53,9 @@ public PojoMassIndexingLoggingMonitor(int logAfterNumberOfDocuments) {
 	@Override
 	public void documentsAdded(long increment) {
 		if ( startTime == 0 ) {
+			// this sync block doesn't seem to require a change:
+			// - sync happens only once at start?
+			// - no I/O and simple in-memory operations
```

Comment:
No, it doesn't happen just once at start. Regardless, I agree it shouldn't be a problem sine we don't use virtual threads in the mass indexer. Here too I think the comment needs some work.

```suggestion
			// this sync block doesn't seem to be a problem for Loom:
			// - always executed in MassIndexer threads, which are not virtual threads
			// - no I/O and simple in-memory operations
```